### PR TITLE
package cleanup

### PR DESCRIPTION
**Pull request by hundeboll-silicom**

### CODE REVIEW

**Review by r-rojo — APPROVED** [review]

File: opae.spec.in [hunk 1]
```diff
@@ -194,6 +196,8 @@ ldconfig
 %dir @CMAKE_INSTALL_PREFIX@/src/opae
 @CMAKE_INSTALL_PREFIX@/src/opae/*
 @CMAKE_INSTALL_PREFIX@/bin/opae.io
+@CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libboard_common.a
```
Comment (tswhison) [inline opae.spec.in:199]: We were told not to package .a files.  It violates some sort of Fedora upstreaming guideline.

File: opae.spec.in [hunk 2]
```diff
@@ -272,6 +275,7 @@ ldconfig
 @CMAKE_INSTALL_PREFIX@/bin/afu_sim_setup
 @CMAKE_INSTALL_PREFIX@/bin/with_ase
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libase*
+@CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libopae-c-ase*
```
Comment (tswhison) [inline opae.spec.in:278]: I think that as of [#116](https://github.com/OPAE/opae-libs/pull/116), the documentation will have to be updated to say that -DBUILD_ASE=yes is always given when the intention is to build RPMs.

File: opae.spec.in (hunk 1 above)
Comment (hundeboll-silicom) [reply opae.spec.in]: You are right:
https://docs.fedoraproject.org/en-US/packaging-guidelines/#packaging-static-libraries

I've pushed new/updated commits where installation of the two static libraries is disabled instead.

File: opae.spec.in (hunk 2 above)
Comment (tswhison) [reply opae.spec.in:276]: Ah yes.. I should have looked at the preceding lines.  Please note that BUILD_ASE should be deprecated in favor of OPAE_BUILD_SIM.  Thanks

File: opae.spec.in [hunk 3]
```diff
@@ -218,6 +220,7 @@ ldconfig
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/opae/libboard_n3000.so*
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/opae/libboard_d5005.so*
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/opae/libboard_a10gx.so*
+@CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/opae/libv-opae.so*
```
Comment (r-rojo) [inline opae.spec.in:223]: This library has been renamed to libopae-v.so

Comment (hundeboll-silicom) [reply opae.spec.in]: Fixed

**Comment by hundeboll-silicom** [comment]
ping

**Comment by hundeboll-silicom** [comment]
@r-rojo @tswhison @anandaravuri please review (again)

I've rebased on current master, and added commits to use `opae_python_install()` for opae.admin and pacsign.

Thanks.

**Review by r-rojo — APPROVED** [review]

File: opae.spec.in [hunk 4]
```diff
@@ -206,13 +198,15 @@ ldconfig
...
 @CMAKE_INSTALL_PREFIX@/bin/afu_json_mgr
+@CMAKE_INSTALL_PREFIX@/bin/bist
 @CMAKE_INSTALL_PREFIX@/bin/bist_app
 @CMAKE_INSTALL_PREFIX@/bin/bist_app.py
 @CMAKE_INSTALL_PREFIX@/bin/bist_common.py
 @CMAKE_INSTALL_PREFIX@/bin/bist_dma.py
 @CMAKE_INSTALL_PREFIX@/bin/bist_def.py
 @CMAKE_INSTALL_PREFIX@/bin/bist_nlb3.py
 @CMAKE_INSTALL_PREFIX@/bin/bist_nlb0.py
+@CMAKE_INSTALL_PREFIX@/bin/coreidle
```
Comment (tswhison) [inline opae.spec.in:209]: Please remove coreidle.  It applies only to a legacy platform that is no longer supported.

Comment (hundeboll-silicom) [reply opae.spec.in]: Done.

**Review by tswhison — DISMISSED** [review]

**Comment by hundeboll-silicom** [comment]
@michelleyho Ping

**Comment by hundeboll-silicom** [comment]
Rebased on current master to fix conflict from https://github.com/OPAE/opae-sdk/pull/1897

**Comment by hundeboll-silicom** [comment]
@michelleyho @r-rojo bump?

**Review by r-rojo — APPROVED** [review]

**Pull request merged**